common/app/views/fragments/header.scala.html

```diff
@@ -59,22 +59,30 @@
                         @fragments.inlineSvg("arrow-right", "icon")
                     </a>
 
-                    <a class="cta-bar__cta hide-until-tablet js-subscribe js-acquisition-link"
+                    @if(editionId != "us" || UsHeaderAndFooterSubscribeLinkSwitch.isSwitchedOn) {
+                        <a class="cta-bar__cta hide-until-tablet js-subscribe js-acquisition-link"
                         data-link-name="nav2 : subscribe-cta"
-                        data-edition="@{editionId}"
+                        data-edition="@{
+                            editionId
+                        }"
                         href="@getReaderRevenueUrl(SupportSubscribe, Header)">
-                        Subscribe
-                        @fragments.inlineSvg("arrow-right", "icon")
-                    </a>
+                            Subscribe
+                            @fragments.inlineSvg("arrow-right", "icon")
+                        </a>
+                    }
 
                     @if(editionId == "uk") {
-                        <a class="cta-bar__cta hide-from-tablet js-change-become-member-link js-acquisition-link"
+                        @if(editionId != "us" || UsHeaderAndFooterSubscribeLinkSwitch.isSwitchedOn) {
```

Comment:
@tomrf1 should this be here, given the switch is only to affect the US edition, I think you could leave this as it was.

Reply:
oops, you're right!

Reply:
It looks to me like Subscribe was only showing for UK edition from tablet upwards.